classes/Tools.php

```diff
@@ -2308,6 +2308,7 @@ public static function generateHtaccess($path = null, $rewrite_settings = null,
         fwrite($write_fd, "AddType font/otf .otf\n");
         fwrite($write_fd, "AddType application/font-woff .woff\n");
         fwrite($write_fd, "AddType font/woff2 .woff2\n");
+        fwrite($write_fd, "AddType image/webp .webp\n");
```

Comment:
```suggestion
        fwrite($write_fd, "AddType image/webp .webp\n");
        fwrite($write_fd, "AddType image/avif .avif\n");
```

I think we should add Avif as well